tests/_utils/test_prepare

```diff
@@ -144,9 +144,9 @@ function run_case() {
 
     eval ${init_table_cmd}
 
-    truncate -s 0 $WORK_DIR/master/log/dm-master.log
-    truncate -s 0 $WORK_DIR/worker1/log/dm-worker.log
-    truncate -s 0 $WORK_DIR/worker2/log/dm-worker.log
+    # truncate -s 0 $WORK_DIR/master/log/dm-master.log
```

Comment:
This is not a good way. Some tests rely on logs(check_log_contains). In shardddl tests, we run different cases in a single cluster, if we keep previous logs, check_log_contains is imprecise.